skbio/stats/composition.py

```diff
@@ -806,6 +807,163 @@ def centralize(mat):
+           Theory Biosci. 135, 21–36 (2016). https://doi.org/10.1007/s12064-015-0220-8
+    """
+    # Convert array_like to numpy array
+    x = np.asarray(x)
+    y = np.asarray(y)
+
+    # Mask zeros
+    if robust:
+        x = np.ma.masked_array(x, mask=x == 0)
+        y = np.ma.masked_array(y, mask=y == 0)
+    
+    # Log transformation
+    x = np.log(x)
+    y = np.log(y)
+    
+    #Variance log ratio
+    return np.var(x - y, ddof=ddof)
+
+@experimental(as_of="0.5.7")
+def pairwise_vlr(mat:Union[np.array, pd.DataFrame], ids:list=None, ddof:int=1, robust=False) -> DistanceMatrix:
+    r"""
+    Performs pairwise variance log ratio transformation.
+    
+    Parameters
+    ----------
+    mat : array_like, float
+       a matrix of proportions where
+       rows = compositions and
+       columns = components
+       
+    ids: array_like, str
+        component names
+        if pd.DataFrame is provided and `ids is None` then `mat.columns` are used for `ids`
+        
+    ddof: int
+        degrees of freedom
+        
+    robust: bool
+        mask zeros at the cost of performance
+       
+    Returns
+    -------
+    skbio.DistanceMatrix
+         distance matrix of variance log ratio values
+    Examples
+    --------
+    >>> # NumPy Array
+    >>> mat = np.asarray([
+        [1,2,3],
+        [5,8,13],
+        [21,34,55],
+    ])
+    >>> dism = pairwise_vlr(mat)
+    >>> dism.redundant_form()
+    # array([[0.        , 0.01576411, 0.00649553],
+    #        [0.01576411, 0.        , 0.00202147],
+    #        [0.00649553, 0.00202147, 0.        ]])
+
+    >>> # Pandas DataFrame
+    >>> df = pd.DataFrame(
+        mat, 
+        index=["sample_1", "sample_2", "sample_3"], 
+        columns=["component_1", "component_2", "component_3"],
+    )
+    >>> dism = pairwise_vlr(df)
+    >>> dism.to_data_frame()
+    # component_1	component_2	component_3
+    # component_1	0.000000	0.015764	0.006496
+    # component_2	0.015764	0.000000	0.002021
+    # component_3	0.006496	0.002021	0.000000
+    
+    References
+    ----------
+    .. [1] V. Lovell D, Pawlowsky-Glahn V, Egozcue JJ, Marguerat S, Bähler J (2015) 
+           Proportionality: A Valid Alternative to Correlation for Relative Data. 
+           PLoS Comput Biol 11(3): e1004075. https://doi.org/10.1371/journal.pcbi.1004075
+    .. [2] Erb, I., Notredame, C. 
+           How should we measure proportionality on relative gene expression data?. 
+           Theory Biosci. 135, 21–36 (2016). https://doi.org/10.1007/s12064-015-0220-8
+    """
+    # Identifiers
+    components = None
+    if isinstance(mat, pd.DataFrame):
+        components = mat.columns
+        mat = mat.values
+    if ids is None:
+        ids = components
+    
+    # Mask zeros
+    X = mat.astype(np.float64)
+    if robust:
+        X = np.ma.masked_array(X, mask=X == 0)
+        
+    # Log Transform
+    X_log = np.log(X)
+    
+    # Variance Log Ratio
+    covariance = np.cov(X_log.T, ddof=ddof) 
```

Comment:
there is a `np.ma.cov` we could replace this with in the `_robust_pairwise_vlr_method`